### DISCUSSION TITLE
assets, composer.json, package.json

### DISCUSSION THREAD

**Discussion by LiamAird** [nWidart/laravel-modules#1542] [Q&A] [answered]
We are wanting to go a more modular way of dividing our application, but we have no intention of wrapping each module as a package, with its own package.json and composer.json packages. 
Also assets, like CSS and images will come from the "base" application and assets will be built with the already existing compiler in the root of the application. 

So the question is, can we strip out all the folders we do not need?
Can we remove composer.json, package.json and vite.config? 

We do tend to keep something generic, like master layouts. Base methods and services.

**Comment by dcblogdev** [comment, marked as answer]
You can remove composer.json, package.json and vite.config but you will need to keep module.json as this is required to load the module's service provider.

**Comment by dcblogdev** [comment]
This is how I normally work, route assets and packages through base Laravel and treat modules like sections to store related code.
